### DISCUSSION TITLE
aerial coverage map

### DISCUSSION THREAD

**Discussion by IfReasonable** [NVlabs/sionna#321] [Q&A] [answered]
Hello, I am new in EM and Raytracing. Now I am trying to use sionna to generate coverage map not only at ground height, but also at low areial height such as at an altitude of 500m. Is it possible to generated such aerial coverage map? A more distant idea is, will it be possible to use sionna to generate 3D coverage maps? :D

**Comment by faycalaa** [comment, marked as answer]
Hi,

You can configure the coverage map's position, orientation, and size using the `cm_center`, `cm_orientation`, and `cm_size` parameters of [Scene.coverage_map()](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Scene.coverage_map).

Concerning your second question, there is currently no plan to implement a 3D equivalent to coverage maps, but you could create multiple coverage maps at different heights.

**Reply by IfReasonable** [reply]
Thank you for your answer!
